README.md

```diff
@@ -1,5 +1,39 @@
+# PageRank by Field
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7458535.svg)](https://doi.org/10.5281/zenodo.7458535)
 
+This repo contains code and data to reproduce the results of the manuscript "The field-dependent nature of PageRank values in citation networks".
+In short, we devise a method to compare PageRanks for papers shared between fields and determine that there is field-specific interest in some papers.
 
-## Database
-The dataset doesn't fit in memory, so the project uses mongoDB.
-Specifically, version 5 was used and was installed on an ubuntu 18 system via [these instructions](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/).
+![field comparison figure](https://raw.githubusercontent.com/greenelab/indices/main/figures/percentile_figure.png)
+
+## Installation
+The Python dependencies for this project are managed via [Conda](https://docs.conda.io/en/latest/miniconda.html).
+To install them and activate the environment, use the following commands in bash:
+
+``` bash
+conda env create --file environment.yml
+conda activate indices 
+```
+
+## Running the pipeline
+The pipeline we used to run our analyses is best suited for running in a cluster environment, because it generates multiple terabytes of intermediate outputs and runs tens of thousands of easily paralellizable processes.
+For an example of how to run the analysis portion of the pipeline on a Slurm cluster, see the `cluster_pipeline/` directory.
+
+The order in which to run the scripts, and a brief description of what the scripts do, can be found below:
```

Comment:
Do you want to describe the Snakemake portion of your pipeline somewhere? Or is that obsolete/not used with the clusterized pipeline?

Reply:
It's not used in the clusterized pipeline, I left it in since it documents how some of the scripts can be run though. I'll add a mention to the README to explain why it's there